src/syrupy/extensions/amber/serializer.py

```diff
@@ -61,7 +61,7 @@ def write_file(cls, snapshot_fossil: "SnapshotFossil") -> None:
                 if snapshot_data is not None:
                     f.write(f"{cls._marker_name} {snapshot.name}\n")
                     for data_line in snapshot_data.splitlines(keepends=True):
-                        f.write(cls.with_indent(data_line, 1))
+                        f.write(cls.with_indent(data_line, 0))
```

Comment:
None snapshot lines should be clearly indicated with a `marker` so this indent is not needed

Reply:
I believe this is to easily differentiate a `#` in a string/snapshot content vs. the snapshot separator. E.g.

```
assert snapshot == "#\n#\n#"
```

makes parsing easier, and I think it also makes it more readable

Reply:
Good point will add a test to capture that